### ISSUE TITLE
Is the accuracy rate Top1 or Top5？

### ISSUE THREAD

**Issue by xxLifeLover** [Sara-Ahmed/SiT#19]
Thank you for your work and sharing. 
I am concerned about whether the accuracy given in the paper is **top1 or top5**, especially the 93.02% obtained by SiT on STL-10 in Table 3.

**Comment by Sara-Ahmed** [comment]
Thanks for your interest in our work. It is top-1 accuracy. We also provide the pre-trained models. Notice that STL-10 provides extra unlabeled data that we used for the SSL training.
